CHANGELOG.md

```diff
@@ -1,7 +1,196 @@
+
+    const panelBefore = (
+        <AccordionItemBody>
+            Voluptate elit eiusmod laborum proident esse officia dolor laboris
+            laboris amet nulla officia cillum.
+        </AccordionItemBody>
+    );
+    ```
+
+    ```tsx
+    // After:
+    import { AccordionItemPanel } from 'react-accessible-accordion';
+
+    const panelAfter = (
+        <AccordionItemPanel>
+            Voluptate elit eiusmod laborum proident esse officia dolor laboris
+            laboris amet nulla officia cillum.
+        </AccordionItemPanel>
+    );
+    ```
+
+3. Remove all instances of `hideBodyClassName`. This prop is no longer valid, as
+   `AccordionItemPanel` components are now hidden without additional styling. If
+   you _must_ have a different `className` prop when an item is collapsed, then
+   you may leverage the new `AccordionItemState` component.
+
+    ```tsx
+    // Before
+    import { AccordionItemPanel } from 'react-accessible-accordion';
+
+    const panelBefore = (
+        <AccordionItemPanel className="foo" hideBodyClassName="foo--hidden" />
+    );
+    ```
+
+    ```tsx
+    // After:
+    import {
+        AccordionItemPanel,
+        AccordionItemState,
+    } from 'react-accessible-accordion';
+
+    const panelAfter = (
+        <AccordionItemState>
+            {({ expanded }) => (
+                <AccordionItemPanel
+                    className={expanded ? 'foo' : 'foo foo--hidden'}
+                />
+            )}
+        </AccordionItemState>
+    );
+    ```
+
+4. Remove all instances of `AccordionItem`’s `expanded` prop and instead use
+   `Accordion`’s `preExpanded` prop. Note that this means that ‘controlled’
+   accordions are no longer a supported pattern. Please raise an issue if you
+   have a use-case which calls for the ability to manually control expanded
+   state.
+
+    ```tsx
+    // Before
+    import { Accordion, AccordionItem } from 'react-accessible-accordion';
+
+    const accordionBefore = (
+        <Accordion>
+            <AccordionItem expanded />
+        </Accordion>
+    );
+    ```
+
+    ```tsx
+    // After:
+    import { Accordion, AccordionItem } from 'react-accessible-accordion';
+
+    const accordionAfter = (
+        <Accordion preExpanded={['foo']}>
+            <AccordionItem uuid="foo" />
+        </Accordion>
+    );
+    ```
+
+5. Remove all instances of `Accordion`’s `accordion` prop. Instead, use a
+   combination of `allowZeroExpanded` and `allowMultipleExpanded` props to suit
+   your requirements. If you were not explicitly setting `accordion` to `false`
+   then you probably are not required to make any changes here.
+
+    ```tsx
+    // Before
+    import { Accordion } from 'react-accessible-accordion';
+
+    const accordionBefore = <Accordion accordion={false} />;
+    ```
+
+    ```tsx
+    // After:
+    import { Accordion } from 'react-accessible-accordion';
+
+    const accordionAfter = <Accordion allowMultipleExpanded />;
```

Comment:
Should the `allowMultipleExpanded` prop here have a value, i.e. `allowMultipleExpanded={true}`?

Reply:
Not out of necessity, no. This is an equivalent shorthand.